### PR TITLE
Dockerize CLI

### PR DESCRIPTION
**Pull request by svenmueller**
@hajoeichler please have a look

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# product type generator
+#
+# VERSION               1.0.0
+
+FROM node:latest
+MAINTAINER Sven Müller <sven.mueller@commercetools.de>
+
+# clone repo
+RUN git clone https://github.com/svenmueller/sphere-product-type-json-generator.git /json-generator
```
Comment (hajoeichler) [inline Dockerfile]: wrong URL

Comment (emmenko) [inline Dockerfile]: Why are you cloning the entire repository?

File: README.md [hunk 2]
```diff
@@ -37,6 +37,19 @@ $ product-type-update
...
+There is also a docker container for easy setup/execution.
+
+Run docker container:
+```bash
+docker run -v /path/to/files/:/files sphereio/product-type-generator
+```
+
+Set an alias for repeated calls:
+```bash
+alias feeder='docker run -v /path/to/files/:/files sphereio/product-type-generator'
```
Comment (emmenko) [inline README.md]: Why is it called `feeder`?

Comment (svenmueller) [inline README.md]: good catch..copy paste ;)

File: Dockerfile (hunk 1 above)
Comment (svenmueller) [inline Dockerfile]: i want to build the latest version whenever someone commits changes

Comment (svenmueller) [inline Dockerfile]: @hajoeichler thx!

**Pull request merged**